tests/test_averager2d.py

```diff
@@ -129,7 +129,7 @@ def _construct_rotations(self):
 
         # Generate rotations to be used by `Simulation`
         self.rotations = Rotation.about_axis(
-            "z", self.thetas, dtype=self.dtype, gimble_lock_warnings=False
+            "z", -self.thetas, dtype=self.dtype, gimble_lock_warnings=False
```

Comment:
Did we ever come to a conclusion on this? As it's just in the test, I'm not super concerned, but it would still be good to have an explanation as to why it flipped.

Reply:
The reason is that Simulation will rotate the grid to produce projections but the averager will rotate the images when aligning with a reference image.

I'll add a comment in the test.

Reply:
Done.